packages/global/templates/website-section/collections.marko

```diff
@@ -41,9 +41,11 @@ $ const queryParams = {
             <if(editorImage)>
               <div>
                 <img src=`${editorImage}` style="height: 150px; width: 150px;" />
-                <if(editorTitle)>
+                <if(editorName)>
                   <p>
-                    <i>${editorTitle}</i>
+                    ${editorName}<br>
+                    ${editorTitle}<br>
+                    <i>${editorSite}</i>
```

Comment:
Instead of applying italics here and splitting this into multiple variables, make it one and utilize break tags (<br>) and just use the Marko HTML render functionality to turn it into HTML inside the paragraph (i.e. $!{editorInfo}). That way if for whatever reason different text formatting needs to be applied it's just changed within the variable over applying it "forcibly" here.